server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/SessionRoutesContract.scala

```diff
@@ -132,6 +132,7 @@ object SessionRoutesContract {
                          |    "urn:ietf:params:jmap:mdn": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6"
                          |  },
                          |  "username" : "bob@domain.tld",
+                         |  "delegatedUsers": [],
```

Comment:
I think we should return delegated accounts in `accounts` properties instead of inventing a new property which not in JMAP spec.
rf: https://jmap.io/spec-core.html#the-jmap-session-resource
https://github.com/apache/james-project/blob/db2f11c4db017f8544c2dac2afe27b75af077841/server/protocols/jmap-rfc-8621/doc/specs/spec/mail/delegation.mdown#addition-to-the-session-object